v0/sources/iterable/transform.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+const fs = require("fs");
+// import mapping json using JSON.parse to preserve object key order
+const mapping = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./mapping.json"), "utf-8")
+);
+const Message = require("../message");
+// const { mappingJson } = require("./mapping");
+
+function process(event) {
+  const message = new Message(`Iterable`);
+
```

Comment:
Put a check if email, eventName and dataFields are there or not else throw error.

Reply:
Done

Reply:
Data fields check is not required some events can have no data fields as rudderstack event property is a free flowing object it can be empty as well

Reply:
Yes we can remove the check for dataFields.

Reply:
Removed check for dataFields